ledger/storage/drop/drop.go

```diff
@@ -17,20 +17,62 @@
 package drop
 
 import (
+	"bytes"
 	"context"
 
+	"github.com/ugorji/go/codec"
+
 	"github.com/insolar/insolar/core"
-	"github.com/insolar/insolar/ledger/storage/jet"
 )
 
-// Modifier provides an interface for modifying jetdrops.
 //go:generate minimock -i github.com/insolar/insolar/ledger/storage/drop.Modifier -o ./ -s _mock.go
+
+// Modifier provides an interface for modifying jetdrops.
 type Modifier interface {
-	Set(ctx context.Context, jetID core.JetID, drop jet.Drop) error
+	Set(ctx context.Context, jetID core.JetID, drop Drop) error
 }
 
-// Accessor provides an interface for accessing jetdrops.
 //go:generate minimock -i github.com/insolar/insolar/ledger/storage/drop.Accessor -o ./ -s _mock.go
+
+// Accessor provides an interface for accessing jetdrops.
 type Accessor interface {
-	ForPulse(ctx context.Context, jetID core.JetID, pulse core.PulseNumber) (jet.Drop, error)
+	ForPulse(ctx context.Context, jetID core.JetID, pulse core.PulseNumber) (Drop, error)
+}
+
+// Drop is a blockchain block.
+// It contains hashes of the current block and the previous one.
+type Drop struct {
+	// nolint: golint
+	// Pulse number (probably we should save it too).
+	Pulse core.PulseNumber
+
+	// PrevHash is a hash of all record hashes belongs to previous pulse.
+	PrevHash []byte
+
+	// Hash is a hash of all record hashes belongs to one pulse and previous drop hash.
+	Hash []byte
+
+	Size uint64
+}
+
+// Encode serializes jet drop.
+func Encode(drop *Drop) ([]byte, error) {
```

Comment:
This code should be without unused error. See the sample here - https://github.com/insolar/insolar/blob/7794e0da5eb975186a4f8dc371e7299493d6fcd4/ledger/storage/object/lifeline.go#L59

Reply:
ok, but it's should be the scope of drop package refactoring